Plugins/IMA/AdEvent.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  AdEvent.swift
+//  Pods
+//
+//  Created by Gal Orlanczyk on 07/03/2017.
+//
+//
+
+import Foundation
```

Comment:
Why move ad events to IMA?
They should be general and implemented as part of other ad plugins.